README.md

```diff
@@ -21,83 +21,83 @@ Sensitive Tree.
+   on style descriptions which describe a subset of a language.
 
 2. Create programs to load these definitions and convert input files
-   into a universal AST, primarily for building a test suite to verify the
-   language definitions are able to parse files at useful levels of detail,
-   again focusing on style-defined subsets of languages which are easier
-   and also more useful.
+   into a universal AST, primarily for building a test suite to verify that
+   the language definitions are able to parse files at useful levels of
+   detail. Again, focusing on style-defined subsets of languages, which are
+   easier and also more useful.
 
-   These programs may use existing parsers, by converting the coAST
+   These programs may use existing parsers by converting the coAST
    definitions into metasyntax used by other parsing toolkits, such as BNF
    and derivatives, Antlr .g4, and augeas.
 
 3. Standardise the definition schema once a sufficiently large number
-   of language definitions have been adequately verified to determine
-   the schema is able to usefully describe most concepts found in
-   commonly used grammars.
+   of language definitions have been verified to determine that
+   the schema can usefully describe most concepts found in
+   commonly-used grammars.
 
 ## Phases
 
-These phases will be overlapping slightly.
+These phases will be slightly overlapping.
 
 ### Phase 1: Replace coala language definitions
 
 The language definitions found at
 https://github.com/coala/coala/tree/master/coalib/bearlib/languages
-will be manually added as language definitions here, growing the schema
-as necessary.  Once the import of facts is complete, a generator will
+will be manually added as language definitions, growing the schema
+as necessary. Once the import of facts is complete, a generator will
 create the coala language definitions from a snapshot of the coAST language
 definitions, putting the collated coAST definitions into use.
 
 ### Phase 2: Import other language definitions
 
 There are many other collections of language definitions.
-Initially the coAST definitions will only link to these external resources,
-and then in the second phase those external grammars will be converted
-into coAST facts, using batch import tools or manually where necessary.
+Initially, the coAST definitions will only link to these external resources.
+In the second phase, those external grammars will be converted
+into coAST facts using batch import tools, or manually where necessary.
 
-In this phase tools to convert the coAST definitions into other syntax
-will be needed, to round trip the language definitions, providing verification
-that the imports are complete, or that partial definitions allow correct
-partial parsing of those languages where complete parsing is too complex.
+In this phase, tools to convert the coAST definitions into other syntax
+will be needed to roundtrip the language definitions. This will provide
```

Comment:
`will be needed to roundtrip the language definitions` - what is `roundtrip` supposed to mean in this phrase?

Reply:
Not entirely certain, I just made it one word.